### PR TITLE
Feature/finding sorting searching

### PR DESCRIPTION
**Pull request by noamblitz**
### Changes

Adds the following features to finding list:

- searching on finding contains
- sorting on either `score` or `finding_type`

### Demo

_Please add some proof in the form of screenshots or screen recordings to show (off) new functionality, if there are interesting new features for end-users._

### QA notes

_Please add some information for QA on how to test the newly created code._

---

### Code Checklist

- [ ] All the commits in this PR are properly PGP-signed and verified.
- [ ] This PR only contains functionality relevant to the issue.
- [ ] I have written unit tests for the changes or fixes I made.
- [ ] I have checked the documentation and made changes where necessary.
- [ ] I have performed a self-review of my code and refactored it to the best of my abilities.

- [ ] Tickets have been created for newly discovered issues.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Review by ammar92 — DISMISSED** [review]
No remarks

**Comment by stephanie0x00** [comment]
## Checklist for QA:

- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
  - [x] I have managed to pass the onboarding flow
  - [x] Objects and Findings are created properly
  - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.
- [x] I checked the logs for errors and/or warnings and made issues where necessary

### What works:
Love this! This makes the user experience so much better. 

### What doesn't work:
- Incase sensitive searching doesn't work. Thus when you have a finding NO-DNSSEC, searching for 'dns' will result in no findings, but searching for DNS will make them show up. 
- The sorting columns do not show up though, as shown below. 


### Bug or feature?:
See feature (request) above. 

### Screenshots: 
The sorting columns are not visible: 
image

File: octopoes/octopoes/repositories/ooi_repository.py [hunk 1]
```diff
@@ -751,6 +757,15 @@ def list_findings(
         elif only_muted:
             muted_clause = "[?muted_finding :MutedFinding/finding ?finding]"
 
+        search_statement = (
+            f"""[?finding :xt/id ?id]
+                                [(clojure.string/includes? ?id \"{escape_string(search_string)}\")]"""
```
Comment (underdarknl) [inline octopoes/octopoes/repositories/ooi_repository.py:762]:
```suggestion
                                [(str/lower-case (clojure.string/includes? ?id \"{escape_string(search_string)}\"))]"""
```

And also lowercase the input search_string

Comment (noamblitz) [reply octopoes/octopoes/repositories/ooi_repository.py:762]: Have you tested that? It doesnt work for me.

Comment (underdarknl) [reply octopoes/octopoes/repositories/ooi_repository.py:762]: Nope, sorry

**Comment by stephanie0x00** [comment]
QA notes:
Helper text is now added to the Findings page with a message that it is case-sensitive. The Object page doesn't have this specific message and the tasks page doesn't have a helper functionality yet. Those are picked up in a different ticket.

**Review by underdarknl — APPROVED** [review]

**Pull request merged**